providers/aws/services/ec2/ec2_service.py

```diff
@@ -0,0 +1,83 @@
+from lib.logger import logger
+import threading
+import urllib.request, json
+from botocore.exceptions import ClientError
+
+from providers.aws.aws_provider import aws_session, audited_account, audited_regions, audited_partition
+
+
+################## EC2
+class EC2:
+    def __init__(self, aws_session, audited_regions):
+        self.service = "ec2"
+        self.aws_session = aws_session
+        self.regional_clients = self.generate_regional_clients(self.service, audited_regions)
+        self.threading_call(self.__describe_snapshots__)
+
+    def __get_clients__(self):
+        return self.clients
+
+    def __get_session__(self):
+        return self.aws_session
+
+    def generate_regional_clients(self, service, audited_regions):
+        regional_clients = []
+        try: # Try to get the list online
+            with urllib.request.urlopen("https://api.regional-table.region-services.aws.a2z.com/index.json") as url:
```

Comment:
Please, include this url under `config`, we cannot hardcode this kind of values.